input/fsh/Profiles clinical datapoints/TumorFactors.fsh

```diff
@@ -0,0 +1,394 @@
+* valueCodeableConcept = SCT#416237000
+
+Mapping: ERStatusToICHOM
+Source:	ERStatus
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: erstatusmapping
+Title: "ER status to ICHOM set"
+Description: "Mapping of the estrogen receptor status to the ICHOM breast cancer PCOM set" 	
+* valueCodeableConcept -> "Estrogen receptor status"
+
+// PR STATUS
+Profile: PRStatus
+Parent: Observation 
+Id: pr-status
+Title: "Progesterone receptor status"
+Description: "Represents if the progesterone receptor status is positive"
+* insert PublicationProfileRuleset
+* code = SCT#445029000 "Status of progesterone receptors of neoplasm (observable entity)"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from ProgesteroneStatusVS (required)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: PRStatusPatient147
+InstanceOf: PRStatus
+Description: "Example of the progesterone receptor status in a patient with breast cancer"
+* status = ObservationStatusCS#final
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = NullFlavor#UNK
+
+Mapping: PRStatusToICHOM
+Source:	PRStatus
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: prstatusmapping
+Title: "PR status to ICHOM set"
+Description: "Mapping of the progesterone receptor status to the ICHOM breast cancer PCOM set" 	
+* valueCodeableConcept -> "Progesterone receptor status"
+
+// HER STATUS
+Profile: HERStatus
+Parent: Observation 
+Id: her-status
+Title: "HER2 receptor status"
+Description: "Represents if the HER2 receptor status is positive"
+* insert PublicationProfileRuleset
+* code = LNC#48676-1 "HER2 [Interpretation] in tissue"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from HER2ReceptorStatusVS (required)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: HERStatusPatient147
+InstanceOf: HERStatus
+Description: "Example of the HER2 receptor status in a patient with breast cancer"
+* status = ObservationStatusCS#final
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = SCT#416237000
+
+Mapping: HERStatusToICHOM
+Source:	HERStatus
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: herstatusmapping
+Title: "HER status to ICHOM set"
+Description: "Mapping of the HER2 receptor status to the ICHOM breast cancer PCOM set" 	
+* valueCodeableConcept -> "HER2 receptor status"
+
+
+// Mammaprint score
+Profile: MammaprintScore
+Parent: Observation 
+Id: mammaprint-score
+Title: "Mammaprint score"
+Description: "Represents the mammaprintscore on a scale of 0.000 to 1.000"
+* insert PublicationProfileRuleset
+* subject only Reference(BreastCancerPatient)
+* value[x] only Quantity
+* value[x] MS
+* effectiveDateTime MS
+// * method --> we would need a code for this
+// can we limit the scale to 0.000 to 1.000? And do we want that?
```

Comment:
We actually can, looks like there is minValue and maxValue available in the FHIR profile - https://www.hl7.org/fhir/elementdefinition-definitions.html#ElementDefinition.minValue_x_

Just need to work out how to set it via FSH and test that it actually works in a validator.

Reply:
Shall I put this on the Jira as something to add later on when we have finished all the profiles?

Reply:
Yes please